packages/core/src/components/plekinfo-card/plekinfo-card.tsx

```diff
@@ -0,0 +1,134 @@
+import {
+  h,
+  Component,
+  ComponentInterface,
+  Element,
+  Event,
+  EventEmitter,
+  Host,
+  Prop,
+  forceUpdate,
+  FunctionalComponent,
+} from "@stencil/core";
+
+import { isInteractiveElement } from "../../utils/is-interactive-element";
+import { isModifiedEvent } from "../../utils/is-modified-event";
+import { DsoPlekinfoCardClickEvent, PlekinfoWijzigactie } from "./plekinfo-card.interfaces";
+
+interface WrapWijzigactieProps {
+  wijzigactie: PlekinfoWijzigactie | undefined;
+}
+
+const WrapWijzigactie: FunctionalComponent<WrapWijzigactieProps> = ({ wijzigactie }, children) => {
+  if (wijzigactie === "voegtoe") {
+    return <ins class="dso-plekinfo-card-container">{children}</ins>;
+  }
+
+  if (wijzigactie === "verwijder") {
+    return <del class="dso-plekinfo-card-container">{children}</del>;
+  }
+
+  return <div class="dso-plekinfo-card-container">{children}</div>;
+};
+
+@Component({
+  tag: "dso-plekinfo-card",
+  styleUrl: "plekinfo-card.scss",
+  shadow: true,
+})
+export class PlekinfoCard implements ComponentInterface {
+  @Element()
+  host!: HTMLDsoPlekinfoCardElement;
+
+  /**
+   * Een optionele wijzigactie die aangeeft of de plekinfo is toegevoegd of verwijderd.
```

Comment:
Nederlandse en Engelse documentatie? 🎏

Reply:
We doen Engels en Nederlands commentaar. Maar  in 1 tsx of alleen Engles of alleen Nedrelands. Is dat correct?

Reply:
Engelse comments.

Gedaan

Reply:
Thanks, hier moeten nog maar eens werkafspraken over maken